### PR TITLE
Change Multimap<Class, Strings> to Map<String, Class>

### PR DESCRIPTION
**Pull request by Crafter-Y**
This resolves issue #12.
Further explanation in there.
If this isn't what is intended for this project, close ths pull.

### CODE REVIEW

**Comment by KevinDaGame** [comment]
Changed base to develop, since Master should only contain releases

File: VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java
Comment (KevinDaGame) [inline VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:162]: I'd personally prefer it if this were to be casted to an int, since there is no reason for this to be a long except that the count() method returns it

File: VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java
Comment (KevinDaGame) [inline VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:145]: here you can do 
```suggestion
       return brushes.get(handle.toLowerCase());
```
that could also get rid of the check on line 126 and the return null on 135

File: VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java
Comment (KevinDaGame) [inline VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:182]:
```suggestion
    public Map<String, Class<? extends IBrush>> getRegisteredBrushesMap() {
```
If you change that, make sure references are changed too

**Review by KevinDaGame — CHANGES_REQUESTED** [review]
Thanks for your contribution. This was indeed our idea with this. Just a few changes though

File: VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java
Comment (Crafter-Y) [reply VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:145]: there is no check to prevent uppercase handle registration (as far as i know). brush in brushes and the handle can be uppercase. So this is still needed?

Comment (KevinDaGame) [reply VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:145]: The method registerSniperBrush puts handle.toLowerCase in the map. And since the map is private we can be sure that there are no uppercase handles

**Comment by KevinDaGame** [comment]
There are two failing unit tests. Could you look into those?

File: VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java
Comment (Lennart99) [inline VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:18]: Do we still need this List? I feel like we could just use brushes.keySet() instead.

**Comment by KevinDaGame** [comment]
Validated ingame, seems to work

File: VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java
Comment (Lennart99) [inline VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:175]: Does this need to be Immutable?

**Comment by KevinDaGame** [comment]
Please squash if you merge

File: VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java
Comment (Crafter-Y) [reply VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:175]: I don't know, if this needs to be immutable, but it was immutable before.
But I think this makes sense, because is sespect, that the contents of the map can be changes from the outside otherwise.

Comment (Lennart99) [reply VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:175]: Yeah, I think it is fine this way.

File: VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java
Comment (Crafter-Y) [reply VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:18]: this is a good idea. I'll to that.

File: VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java
Comment (Crafter-Y) [reply VoxelSniperCore/src/main/java/com/github/kevindagame/VoxelBrushManager.java:172]: it is, but it isn't a Multimap

**Comment by Lennart99** [comment]
Would probably be good to run through the javadocs on this file before merging, just to ensure everything is still correct.

**Comment by Crafter-Y** [comment]
there should be averyhing right now

**Comment by Crafter-Y** [comment]
sorry, cannot figure out how to squash these commits here

**Comment by Lennart99** [comment]
builds need to finish first

**Comment by KevinDaGame** [comment]
> sorry, cannot figure out how to squash these commits here

Next to the merge button there should be an arrow, if you click that you can select squash

**Comment by Crafter-Y** [comment]
Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

**Comment by KevinDaGame** [comment]
Ah I see, then I will merge it once checks have been completed

**Comment by KevinDaGame** [comment]
If you'd like to contribute to more issues, please join our discord so we can communicate more effectively

**Comment by Crafter-Y** [comment]
yeah, maybe another day

**Pull request merged**
